library/blivet.py

```diff
@@ -258,6 +258,19 @@ def _get_device_id(self):
     def _type_check(self):
         return self._device.is_disk
 
+    def _look_up_device(self):
+        super(BlivetDiskVolume, self)._look_up_device()
+        if not self._get_device_id():
+            # FAIL: no disks specified for volume
```

Comment:
The comment can be removed, it contains the same text as the exception in the next line